File: 1468-CheckIfNAndItsDoubleExist/1468-CheckIfNAndItsDoubleExist.js
// Last updated: 4/29/2026, 10:24:12 AM
/**
 * @param {number[]} arr
 * @return {boolean}
 */
var checkIfExist = function(arr) {
    for(let i = 0; i < arr.length - 1; i++) {
        for(let j = i+1; j < arr.length; j++) {
            if(arr[i] === 2*arr[j] || arr[j] === 2 * arr[i]) {
                return true;
            }
        }
    }
    return false;
};

File: 1468-CheckIfNAndItsDoubleExist/1468-CheckIfNAndItsDoubleExist.py
# Last updated: 4/29/2026, 10:24:07 AM
class Solution:
    def checkIfExist(self, arr: List[int]) -> bool:
        n = len(arr)
        for i in range(n-1):
            for j in range(i+1, n):
                if arr[i] == 2*arr[j] or arr[j] == 2*arr[i]:
                    return True
        return False